### ISSUE TITLE
Add random generated data from schema

### ISSUE THREAD

**Issue by crowz4k** [boyney123/mockit#41]
It would be great if we can provide JSON schema and API generates data for us.

**Comment by boyney123** [comment]
Hey @crowz4k 

I like this idea, I will try and take a look into it to see what I can do.

I think I have a few ideas.

Is this to change the behavior of the randomly generated data?

**Comment by crowz4k** [comment]
Yes, so you can provide a schema like JSON and then you can get randomly generated data and that data can be generated either automatically or when the user hit the route. 

Why this is good?
Let's say you want to test system where every time you need to have different data, so if you have static data it will not work. But if you have the option to get random each time you hit a route that will be awesome.
